contracts/src/v0.8/dev/ocr2dr/OCR2DROracleFactory.sol

```diff
@@ -31,6 +31,7 @@ contract OCR2DROracleFactory is TypeAndVersionInterface {
    */
   function deployNewOracle() external returns (address) {
     OCR2DROracle oracle = new OCR2DROracle();
+    oracle.transferOwnership(msg.sender);
```

Comment:
It's worthwhile noting that this doesn't fully transfer ownership, as the `msg.sender` (in this case) needs to accept the transfer of ownership after this.

Reply:
Added to the function signature doc!